Extension/c_cpp_properties.schema.json

```diff
@@ -72,12 +72,15 @@
                         }
                     },
                     "intelliSenseMode": {
-                        "description": "If set, it overrides the default mode used by the IntelliSense engine. Windows defaults to msvc-x64, Linux defaults to gcc-x64, and Mac defaults to clang-x64.",
+                        "description": "The IntelliSense mode to use that maps to an architecture-specific variant of MSVC, gcc, or Clang. If not set or if set to ${default}, the extension will choose the default for that platform. Windows defaults to msvc-x64, Linux defaults to gcc-x64, and macOS defaults to clang-x64. Select a specific IntelliSense mode to override the ${default} mode.",
```

Comment:
I would remove the last line. `Select a specific IntelliSense mode to override the ${default} mode.`